cmd/multifile/manifest.go

```diff
@@ -61,21 +60,9 @@ func checkSeenDests(seenDests map[string]string, dest string, urlString string)
 	return nil
 }
 
-func addEntry(manifestMap pget.Manifest, schemeHost string, urlString string, dest string) pget.Manifest {
-	entries, ok := manifestMap[schemeHost]
-
-	if !ok {
-		manifestMap[schemeHost] = []pget.ManifestEntry{{URL: urlString, Dest: dest}}
-	} else {
-		manifestMap[schemeHost] = append(entries, pget.ManifestEntry{URL: urlString, Dest: dest})
-	}
-
-	return manifestMap
-}
-
 func parseManifest(file io.Reader) (pget.Manifest, error) {
-	seenDests := make(map[string]string)
-	manifestMap := make(pget.Manifest)
+	seenDestinations := make(map[string]string)
+	manifest := make(pget.Manifest, 0)
```

Comment:
```suggestion
	manifest := make(pget.Manifest)
```
think this was added by mistake?

Reply:
ah I missed the type had changed from map to slice